packages/strapi-plugin-content-manager/admin/src/components/SelectMany/ListItem.js

```diff
@@ -18,7 +20,19 @@ function ListItem({
   onRemove,
   targetModel,
 }) {
-  const to = `/plugins/${pluginId}/collectionType/${targetModel}/${data.id}`;
+  const { settingsBaseURL } = useGlobalContext();
+  const to = useMemo(() => {
+    const isSettingsModel = targetModel.includes('strapi::');
+
+    if (isSettingsModel) {
+      const model = pluralize(targetModel.replace('strapi::', ''));
+
+      return `${settingsBaseURL}/${model}/${data.id}`;
+    }
+
+    return `/plugins/${pluginId}/collectionType/${targetModel}/${data.id}`;
+  }, [targetModel, data.id, settingsBaseURL]);
+
```

Comment:
I think it is ok to have custom code for this functionnality. What do you think if the link is enabled only for `strapi::user` and disabled for the other `strapi::user` ? The pluralize would not be needed in that case

Reply:
The pluralize is needed in order to match the correct endpoint in the settings view.

Reply:
I have a made a modification.